lucida/weather/WeatherConfig.py

```diff
@@ -0,0 +1,15 @@
+"""
+Weather API configuration details
+"""
+
+PORT = 8088
+
+# Weather Underground API key
+# https://www.wunderground.com/weather/api/
+WU_API_URL_BASE = 'http://api.wunderground.com/api/'
+WU_API_KEY = 'ff76e8e80c802d46' # TODO: remove after PR has been accepted # add your API key here
+
+# Open Weather Map API key
+# https://openweathermap.org/api
+OWM_API_URL_BASE = 'http://api.openweathermap.org/data/2.5/weather?'
+OWM_API_KEY = '362537b891e6df03a316e82565fe4df3' # TODO: remove after PR has been accepted # add your API key here
```

Comment:
Noted. have your own branch that across all the MRs applies your keys. keep rebasing and applying that branch to master.